UI/src/view/assetSelectionMenu.js

```diff
@@ -1,34 +1,69 @@
 (function (root) {
-  root.AssetSelectionMenu = function(assetGroups, options) {
+  root.AssetSelectionMenu = function(linearAssetGroups, pointAssetGroups, maintenanceAssetGroup, options) {
     var onSelection = options.onSelect;
-    var assetSelection = $('<div class="asset-selection"></div>');
-
-    var assetLinks =
-      _.chain(assetGroups)
-        .map(function(assets) {
-          var assetGroupDiv = $('<div class="asset-group"></div>');
-          return assetGroupDiv.append(_.map(assets, function (asset) {
-
-              var radioDiv= $($('<div class="radio">').append($('<label>', {
-                'for': 'nav-' + asset.layerName,
-                text: ' ' + asset.title
-              }).prepend($('<input>', {
-                type: 'radio',
-                name: 'navigation-radio',
-                id: 'nav-' + asset.layerName,
-                value: asset.layerName
-              }))));
-            if (asset.title == "Rautateiden huoltotie") {
-              radioDiv.prepend($('<span><b>' + "Vain ELYn ylläpidossa" + '</b></span>'));
-            }
-            return radioDiv;
-          }));
-        })
+    var assetSelection = $('<div class="asset-selection">' +
+      '<div class="column-1"></div>' +
+      '<div class="column-2"></div>'+
+      '</div>');
+
+    var column1 =  $('<div class="column-1"></div>');
+    var column2 = $('<div class="column-2"></div>');
```

Comment:
Unused variables, since both div's are defined in the assetSelection variable above.